src/styles/main.css

```diff
@@ -1,62 +1,142 @@
 /* styleint-disable */
-.eachtask {
+
+@import url("https://fonts.googleapis.com/css2?family=Courgette&family=Josefin+Slab:ital,wght@0,300;0,400;0,700;1,500&family=Patrick+Hand&family=Rubik:wght@400;500;700&display=swap");
+
+* {
```

Comment:
- [ ] As requested in the first comment, remove the `/* styleint-disable */` comment for style too. This will help you to be sure that you're following some industry-world best practices by fixing the linter errors. Kindly, fix these issues.